src/getCoinCombination.test.js

```diff
@@ -7,4 +7,20 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('Should return [1, 0, 0, 0] when given 1 cent', () => {
```

Comment:
I would write something like that:
`should return '1' pennie when '1' is entered`
But it is just another option on how you can write tests.